parser-typechecker/tests/Unison/Test/Git.hs

```diff
@@ -35,6 +36,7 @@ test = scope "git" . tests $ [testPull, testPush]
 -- dependencies
 testPull :: Test ()
 testPull = scope "pull" $ do
+  branchCache <- io $ Branch.boundedCache 4096
```

Comment:
Was there some reasoning behind the number 4096?

Reply:
Seemed like a nice round number. Bigger seemed like a waste, smaller seemed like not enough

Reply:
Also base last I checked had like 7-10k namespace files. I suspect it's not important to have all the namespace files cached, just "recent history" needed to resolve LCAs of merges etc.